homeassistant/components/synology_dsm/binary_sensor.py

```diff
@@ -14,6 +14,8 @@
     SYNO_API,
 )
 
+DEFAULT_DEVICE_CLASS = "safety"
```

Comment:
Please import and use the device class constant from the binary sensor integration.
https://github.com/home-assistant/core/blob/b0a2c8d430e008a29a84676585c5593a8790e88b/homeassistant/components/binary_sensor/__init__.py#L83

Reply:
```suggestion
DEFAULT_DEVICE_CLASS = DEVICE_CLASS_SAFETY
```